designs/dragonphy_top/construct.py

```diff
@@ -165,6 +175,8 @@ def construct():
         'analog_core.spi',
         'mdll_r1_top_macro.cdl',
         'input_buffer.spi',
+        'input_divider.spi',  # TODO: should this be *.lvs.v instead?
```

Comment:
@sjkim85 what is the correct extension for these netlists, which are used for LVS?

Reply:
@sgherbst , .spi is correct for both phase_interpolator and input_divider.